RecoLocalCalo/CaloTowersCreator/python/calotowermaker_cfi.py

```diff
@@ -153,3 +155,10 @@
                       HEDThreshold1 = cms.double(0.1),
                       HEDThreshold  = cms.double(0.2)
 )
+
+# needed to handle inner/outer and 28/29 splitting w/ collapsed rechits
+from Configuration.ProcessModifiers.run2_HECollapse_2018_cff import run2_HECollapse_2018
+run2_HECollapse_2018.toModify(calotowermaker,
+    HcalPhase = cms.int32(0),
+    HcalCollapsed = cms.bool(True),
```

Comment:
Hi @kpedro88 
since we now give hbhereco as input to the calotowermaker, we want to use old thresholds in the collapsed case.
I think the following should be added in this modification function:

```
HESThreshold1 = cms.double(0.8),
HESThreshold  = cms.double(0.8),
HEDThreshold1 = cms.double(0.8),
HEDThreshold  = cms.double(0.8)
```